libs/kg-commons/src/main/java/eu/ebrains/kg/commons/jsonld/JsonLdDoc.java

```diff
@@ -222,4 +223,48 @@ private void replaceNamespace(String oldNamespace, String newNamespace, Map curr
             }
         }
     }
+
```

Comment:
We should move this logic to another place - the query validation is not a generic "JsonLdDoc" logic. It could e.g. be moved to the "KgQuery" class.